contrib/vagrant/provision-config.sh

```diff
@@ -7,14 +7,14 @@ set -o pipefail
 # Set a useful format for xtrace output
 # export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
-ORIGIN_ROOT=$(
+OS_ROOT=$(
```

Comment:
See any test_networking run in the last few hours.

On May 11, 2016, at 6:57 PM, Steve Kuznetsov <notifications@github.com>
wrote:

In contrib/vagrant/provision-config.sh
<https://github.com/openshift/origin/pull/8743#discussion_r62941230>:

> @@ -7,14 +7,14 @@ set -o pipefail
>  # Set a useful format for xtrace output
>  # export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
>
> -ORIGIN_ROOT=$(
> +OS_ROOT=$(

@marun <https://github.com/marun>

In this PR I had changed this to reflect the $OS_ROOT name used in all
other Bash scripts -- if @smarterclayton <https://github.com/smarterclayton>
is correct and this change is what caused the networking test to fail, can
we change this? Who depends on this name?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8743/files/eb574408ad3142860f722245dd1f72cc7a9369ba#r62941230>